ci/docker/concourse-runner/Dockerfile

```diff
@@ -52,6 +54,12 @@ RUN echo "$FLY_CLI_SHA256SUM  /tmp/fly" >> /tmp/fly.sha256 \
   && mv /tmp/fly /usr/local/bin/ \
   && chmod 555 /usr/local/bin/fly
 
+RUN wget "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" \
```

Comment:
I hate this, but I also know it's the official way to install in ubuntu/debian :( 
I wish they at least published hashes for them so you could validate :(